internal/cli/atlas/clusters/clusters.go

```diff
@@ -12,30 +12,32 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package atlas
+package clusters
```

Comment:
moved cluster to its own package, may do a separate PR if it makes easier to review this one

Reply:
👍